cliboa/scenario/load/gcp.py

```diff
@@ -570,3 +570,68 @@ def execute(self, *args):
                 fname = os.path.splitext(os.path.basename(file))[0]
                 doc = firestore_client.collection(self._collection).document(fname)
                 doc.set(json.load(f))
+
+class BigQueryCopy(BaseBigQuery):
+    """
+    Copy Bigquery from one table to another 
+    """
+
+    def __init__(self):
+        super().__init__()
+
+        self._dest_dataset = None
+        self._dest_tblname = None
+
+    def dest_dataset(self, dest_dataset):
+        self._dest_dataset = dest_dataset
+
+    def dest_tblname(self, dest_tblname):
+        self._dest_tblname = dest_tblname
+
+    def execute(self, *args):
+        super().execute()
+        valid = EssentialParameters(
+            self.__class__.__name__, [
+                self._dataset,
+                self._tblname,
+                self._location,
+                self._dest_dataset,
+                self._dest_tblname
+            ]
+        )
+        valid()
+
+        # Read Credentials
+        if isinstance(self._credentials, str):
```

Comment:
Other classes support string type to ensure backward compatibility, but it is not necessary for newly created class.

Reply:
Thanks for the review.
Fixed it!